### PR TITLE
CORE: View an individual article

### PR DESCRIPTION
**Pull request by tonymm55**
This Pull Request is for  the addition of a singleArticleView & ArticleCard, and related to task 5, "View an individual article".

### CODE REVIEW

File: api.jsx [hunk 1]
```diff
@@ -16,4 +16,27 @@ export const fetchArticles = async (topic) => {
   }
 };
 
-export default { fetchArticles };
+export const fetchArticleById = async (articleId) => {
+  try {
+    const response = await ncNewsAPI.get(`/articles/${articleId}`);
+    console.log("GET request successful, response:", response);
```
Comment (nc-bela-bertalan) [inline api.jsx:22]: Please remove all debugging logs.

File: api.jsx [hunk 2]
```diff
@@ -16,4 +16,27 @@ export const fetchArticles = async (topic) => {
...
+    console.error("Error data: ", error.response.data);
+    return await Promise.reject(error);
+  }
+};
+
+export default { fetchArticles, fetchArticleById };
+
+//Endpoints--------//
+
+// app.get("/api/topics", getAllTopics);
```
Comment (nc-bela-bertalan) [inline api.jsx:35]: These comments shouldn't be here in this PR.

File: src/Components/SingleArticleView.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import { fetchArticleById } from "../../api";
+import { useState, useEffect } from "react";
+
+function SingleArticleView() {
+  const { articleId } = useParams();
+  const [article, setArticle] = useState(null);
```
Comment (nc-bela-bertalan) [inline src/Components/SingleArticleView.jsx:8]: Please set up a loading state and conditionally render a message to the user while the article is loading.

**Review by nc-bela-bertalan — COMMENTED** [review]
Hey Anthony, nice work on this one. Please see my comments.

File: api.jsx (hunk 1 above)
Comment (tonymm55) [reply api.jsx:22]: Thought I had!!

File: api.jsx (hunk 2 above)
Comment (tonymm55) [reply api.jsx:35]: OK, deleted.

File: src/Components/SingleArticleView.jsx (hunk 3 above)
Comment (tonymm55) [reply src/Components/SingleArticleView.jsx:8]: Will do, thanks for the advice Bela.

**Pull request merged**